From 8443cbedaca43eff72a17f98465475aa7f49d445 Mon Sep 17 00:00:00 2001
From: Zheng Zengkai <zhengzengkai@huawei.com>
Date: Thu, 23 Dec 2021 21:31:52 +0800
Subject: [PATCH] arm64: openeuler_defconfig: increase CONFIG_NR_CPUS value to
 4096
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 8443cbedaca43eff72a17f98465475aa7f49d445


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I4J96Y
CVE: NA

-------------------------------------------------

Set CONFIG_NR_CPUS of openeuler_defconfig to 4096 to meet
scalability requirement for arm64.

Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Reviewed-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 73cd7d9c9a78..66124b991d96 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -380,7 +380,7 @@ CONFIG_ARM64_PA_BITS=52
 CONFIG_CPU_LITTLE_ENDIAN=y
 CONFIG_SCHED_MC=y
 CONFIG_SCHED_SMT=y
-CONFIG_NR_CPUS=1024
+CONFIG_NR_CPUS=4096
 CONFIG_HOTPLUG_CPU=y
 # CONFIG_ARM64_BOOTPARAM_HOTPLUG_CPU0 is not set
 CONFIG_NUMA=y
-- 
2.26.2

